examples/routing_service/routing_service_with_security/securedPeerQosProfile.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+
+<dds xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="http://community.rti.com/schema/6.0.0/rti_dds_qos_profiles.xsd"
+    version="6.0.0">
+
+    <qos_library name="SecurityExampleProfiles">
+    
+        <!-- 
+        Profile 'SecuredPeer' is based on the security profile.
+        -->
+        <qos_profile name="SecuredPeer" base_name="BuiltinQosLib::Generic.Security" is_default_qos="true">
+            <participant_qos>
+                <property>
+                    <value>
+                        <element>
+                            <name>dds.sec.auth.identity_ca</name>
+                            <value>file:../../../resources/certAuthority/dsa01/ca/dsa01RootCaCert.pem</value>
```

Comment:
ecdsa01

Reply:
Corrected path. I have also added for `rsa01` and `dsa01`, which are commented.